src/components/Header/NotificationsToggle.ts

```diff
@@ -0,0 +1,107 @@
+import { LitElement, css, html } from "lit";
+import { customElement, state } from "lit/decorators.js";
+import { unsafeHTML } from "lit/directives/unsafe-html.js";
+import { localized, msg } from "@lit/localize";
+import bellIcon from "../../assets/icons/bell.svg?raw";
+import { DropdownController } from "../../controllers/dropdown.ts";
+import { getEnvSettings } from "../../env-settings.ts";
+import {
+  NotificationDataEntry,
+  fetchNotifications,
+} from "../../utils/fetch.ts";
+import { theme } from "../../utils/theme.ts";
+
+const envSettings = getEnvSettings();
+
+@customElement("bkd-notifications-toggle")
+@localized()
+export class NotificationsToggle extends LitElement {
+  @state()
+  notifications: ReadonlyArray<NotificationDataEntry> = [];
+
+  static styles = [
+    theme,
+    css`
+      button {
+        cursor: pointer;
+        border: none;
+        background: transparent;
+        padding: 0;
+        display: flex;
+      }
+
+      .circle {
+        color: var(--bkd-func-bg-white);
+        background-color: var(--bkd-brand-red);
+        box-shadow: 0 0 0 2px var(--bkd-func-bg-white);
+        border-radius: 50%;
+        font-weight: 700;
+        font-size: small;
```

Comment:
- Ich hätte hier ein konkreter `rem` Wert verwendet statt `small`
- Und dann stellt sich immer die Frage ob `px` oder `rem` – ich hätte hier wohl die `width` und `height` auch in `rem` definiert, da der Kreis ja mit der Font «mitskalieren» müsste.